src/components/Compose/ComposeUI.tsx

```diff
@@ -45,14 +46,14 @@ export const ComposeUI = memo(function ComposeUI() {
 
     const addImages = useAddImages();
     const addVideo = useAddVideo();
-    const handleDropFiles = useCallback(
-        (files: File[]) => {
+    const [, handleDropFiles] = useAsyncFn(
```

Comment:
Present loading indicator if loading is true.

```suggestion
    const [{ loading }, handleDropFiles] = useAsyncFn(
```